python/cudf/bindings/quantile.pyx

```diff
@@ -43,6 +43,9 @@ def apply_quantile(column, quant, method, exact):
     Returns value with the quantile specified by quant
     """
 
+    if len(column) == 0:
```

Comment:
Whether you have this code or not, we need to fix this on the libcudf side for consistency across language interfaces. libcudf should return an empty column rather than raising an error.